### ISSUE TITLE
Duplicates and watched in same sub-catagory.

### ISSUE THREAD

**Issue by Ratteler** [yuliskov/SmartTubeNext#2815]
I'm see double and triple of the same video in all sub categories.
I also have zero interest in watching videos I've already watched.

Can something be done so videos only appear once in any category.
I would also like Recently uploaded to be the first category.

**Comment by Fire-** [comment]
Agree, it seems like more and more of my shelves, including recommended, is populated with videos I've already watched ( fully! )

**Comment by lee3lions** [comment]
I’m seeing duplicates and even triplicate videos in the same category. If I remove one with ‘not interested’, the others close too. This has been happening for a while now.
